tests/components/http/test_view.py

```diff
@@ -34,9 +35,16 @@ async def test_invalid_json(caplog):
     view = HomeAssistantView()
 
     with pytest.raises(HTTPInternalServerError):
-        view.json(float("NaN"))
+        view.json("\ud800")
```

Comment:
The actual test runner might be a bit too helpful here as it seems like its detecting the invalid data before it gets to where it needs to go